src/FormField/Radio.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
http://stackoverflow.com/questions/26550745/correct-syntax-for-inheritdoc-in-phpdocumentor

I'm adding @inheritDoc , hopefully that helps.